api/search.js

```diff
@@ -7,18 +9,21 @@ const API_PORT = process.env.API_PORT;
  * @param {Object} cookies is a cookie object stored in the local storage when use set their location.
  * @returns an Array of JSON containing search results from all three serviced i.e. Postmates, Grubhub, and DoorDash.
  */
-const search = async (searchInput, cookies) => {
-  return await (
+const search = async (searchInput) => {
+  const cookies = await getLocalStorage("cookies");
+  const res = await (
     await fetch(`http://${API_DOMAIN}:${API_PORT}/api/search`, {
       method: "POST",
       headers: { "Content-Type": "application/json" },
       credentials: "include",
       body: JSON.stringify({
         query: searchInput,
-        cookies: cookies.cookies,
+        cookies: (await getLocalStorage("cookies")).cookies,
```

Comment:
You already have the cookie data in line 13

`const cookies = await getLocalStorage("cookies");`

So instead of calling the getLocalStorage function again, we can just use that data:

```javascript
      body: JSON.stringify({
        query: searchInput,
        cookies: cookes.cookies})
```

Reply:
Done!